events/hfe_events_maintenance.txt

```diff
@@ -29,7 +29,7 @@ ship_event = {
 				hfe_effect_convert_to_asteroid = yes
 			}
 		}
-		#log = "drone loss starbase on [root.solar_system.GetName] killer [root.from.owner.GetName]"
+		log = "drone loss starbase on [root.solar_system.GetName] killer [root.from.owner.GetName]"
```

Comment:
I dont think spamming in logs is a good idea
Should you add global flag

Reply:
Okay.